### PR TITLE
keep to c3 wo and fc only

### PR DESCRIPTION
**Pull request by pindge**
For paul's visibility.

### CODE REVIEW

**Review by whatnick — CHANGES_REQUESTED** [review]
We should aim for a single, refactored ows_cfg object. With subsets of layers created in separate python files with the existing ows_cfg aggregating them.

File: dev/services/wms/ows/c3_ows_cfg.py [hunk 1]
```diff
@@ -0,0 +1,469 @@
...
+            #     "abstract": "Wet or Sea",
+            #     "flags": {"or": {"wet": True, "sea": True}},
+            #     "color": "#4F81BD",
+            # },
+        ]
+    },
+}
+
+
+ows_cfg = {
```
[the remark this answers is not in the document]
Comment (pindge) [reply dev/services/wms/ows/c3_ows_cfg.py:151]: @SpacemanPaul , is now the perfect opportunity to build ows with fragmented ows_cfg files?

File: dev/services/wms/ows/c3_ows_cfg.py [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+            #     "flags": {
+            #         "dry": True,
+            #         "sea": False,
+            #     },
+            #     "color": "#D99694",
+            # },
+            # {
+            #     "title": "Wet",
+            #     "abstract": "Wet or Sea",
+            #     "flags": {"or": {"wet": True, "sea": True}},
```
Comment (pindge) [inline dev/services/wms/ows/c3_ows_cfg.py:86]: @GypsyBojangles I need some advice on what to do with `sea` flag

File: dev/services/wms/ows/c3_ows_cfg.py [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+        {
+            "flags": {"dry": True},
+        },
+        {
+            "flags": {
+                # "terrain_or_low_angle": False,
+                "high_slope": False,
+                "cloud_shadow": False,
+                "cloud": False,
+                # "sea": False,
```
Comment (pindge) [inline dev/services/wms/ows/c3_ows_cfg.py:36]: @GypsyBojangles this fc style with `pq_masks` enabled doesn't show dense green anymore, which looks very similar to fc_albers, i will need to look at the coastal areas to see how much impact it is without `sea` flag

File: dev/services/wms/ows/c3_ows_cfg.py (hunk 2 above)
Comment (GypsyBojangles) [reply dev/services/wms/ows/c3_ows_cfg.py:86]: To summarize our conversation with Team X:
- When we replace the old flag "terrain_or_low_angle", we should replace with both "low_solar_angle" and "terrain_shadow". Lets start by assumign that these are both set to whatever terrain_or_low_angle was
- For masking the sea bit away, use Fractional Cover Percentiles as a base:
                    "flags": {
                        "band": "land",
                        "product": "geodata_coast_100k",
                        "ignore_time": True,
                        "ignore_info_flags": [],
                    },

**Comment by pindge** [comment]
Merged in through refactor

**Pull request closed without merging**